modules/RTC/TraceablePeerConnection.js

```diff
@@ -417,33 +417,37 @@ function extractSSRCMap(desc) {
         return;
     }
 
-    session.media.forEach(function (bLine) {
-        if (!Array.isArray(bLine.ssrcs))
+    session.media.forEach(function (mLine) {
+        if (!Array.isArray(mLine.ssrcs))
         {
             return;
         }
 
-        if (typeof bLine.ssrcGroups !== 'undefined' &&
-            Array.isArray(bLine.ssrcGroups)) {
-            bLine.ssrcGroups.forEach(function (group) {
+        if (typeof mLine.ssrcGroups !== 'undefined' &&
```

Comment:
also in many places i think we (or at least i) have just done `if (mLine.ssrcGroups)`, since undefined resolves to falsey.  checking if it's actually an array is going the extra mile, but, technically then we should be checking all types are the ones we expect :)